### ISSUE TITLE
Can Bindiff compare resources?

### ISSUE THREAD

**Issue by karpiyon** [google/bindiff#59]
I have 2 binaries that are different in the resource section.
E.g. one has:
...
.rdata:000000014064D9E0                 text "UTF-16LE", '**Some** text or variables',0
.rdata:000000014064D9EE                 align 10h
...
and the other has:
.rdata:000000014064D9E0                 text "UTF-16LE", '**Other** text and variables',0
.rdata:000000014064D9EE                 align 10h
...

Can Bindiff detect these changes?
Of course this is not pure text comparison as these are related to functions and referenced in teh code.

**Comment by cblichmann** [comment]
No, BinDiff does not look at resources. It will only consider text sections (or rather their disassembly).
